### PR TITLE
Adding sass variants of the less files

### PR DESCRIPTION
**Pull request by dmoojunk**
I will confirm this works 100% as expected in work tomorrow, but it compiles with compass and looks the part. Did some light testing tonight and all seems well. 

Any interest in publishing this in an update to npm if you see fit to merge?

Thanks for the library! I had never done 'fancy' scss before and your less was a great codebase to dabble with.

### CODE REVIEW

**Comment by dmoojunk** [comment]
Rolled it into the work build and looks good :-)

**Comment by leejordan** [comment]
This looks great thanks very much for taking the time to do this. I need to do a new release as it's long overdue but I will try and get to it as soon as possible.

File: scss/reflex/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+}
+
+@mixin calc-reflex-columns($index, $class, $type) {
+    @if $type == width and $index > 0 {
+        .#{$reflex-prefix}#{$class}#{$index} {
+            width: percentage(($index / $reflex-columns));
+            *width: percentage(($index / $reflex-columns)) - 0.1;
+
+            //for ie6 support you can uncomment this line but it will increase css filesize dramatically
+            //.setupCols();
```
Comment (dmoojunk) [inline scss/reflex/_mixins.scss]: This line actually needs updating to @include setupCols()

**Comment by iamstuartwilson** [comment]
@dmoojunk this is top work!  You also need to remove [my commit](https://github.com/dmoojunk/reflex/commit/9d851194e132a34b3ba620673248354693fb264b#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) to ensure that your great `.scss` does not get overwriteen by `npm run build` as it does currently.

Easiest way would be to do:

```
git checkout d61d8f0ed14637440ec088a77f0ceae68247d702 package.json
```

## Compilation diffs for reference:

**Ruby Sass (default)**
https://www.diffchecker.com/dcjxhiec
Some added lines, no change to actual CSS

**Node Sass**
https://www.diffchecker.com/zkgmvd4x
No change

Seriously, awesome @dmoojunk @leejordan  :+1:

**Comment by dmoojunk** [comment]
Thanks! I was going to ask about the npm script. Ill roll in a sass compilation step tomorrow evening and update the PR.

**Comment by leejordan** [comment]
@dmoojunk You probably don't need to worry about adding a SASS compilation step as it will just build the same css file as the LESS compilation. I guess you could add a separate SASS build option to npm scripts though if you wanted. I use the LESS one locally myself.

It's up to you but perhaps just back out @iamstuartwilson's commit for now and I'll merge and sort out a new release.

**Comment by dmoojunk** [comment]
@leejordan @iamstuartwilson I think this is what you had in mind? I fixed forward as frankly my git branch management skills have some room for improvement.

**Comment by iamstuartwilson** [comment]
@dmoojunk @leejordan works a treat :+1:

**Pull request merged**